### ISSUE TITLE
error parsing mapping JSON: unexpected end of JSON input mapping contents:

### ISSUE THREAD

**Issue by guycalledseven** [ponzu-cms/ponzu#162]
If structure of generated item changes, ponzu will fail while initialising bleve with error
 ```
error parsing mapping JSON: unexpected end of JSON input
mapping contents:
```

Solution is to delete search index from `search/<Item>` folder.

Maybe this could be handled better?

**Comment by guycalledseven** [comment]
Haha, yes, float64 was in my case. :) I tried making own `SearchMapping` but that did not have any effect.

Renaming, adding and removing fields and content types from db would be really awesome. In a week of usage - I needed to insert, delete and rename content types and fields in bolt db. In prototyping phase I would really like to be able to do that easier.

However, db migration on that level is pretty complex thing to do automagically (just by analysing source vs data).

Possible approach:

`ponzu generate add-field <new field>` - adds field to db and file
`ponzu generate remove <content type>` - deletes data and file?
